python/pylibraft/pylibraft/common/interruptible.pxd

```diff
@@ -22,11 +22,11 @@
 from libcpp.memory cimport shared_ptr
 from rmm._lib.cuda_stream_view cimport cuda_stream_view
 
-cdef extern from "raft/interruptible.hpp" namespace "raft" nogil:
+cdef extern from "raft/core/interruptible.hpp" namespace "raft" nogil:
```

Comment:
Were these just errors before? It looks like one file just includes the other, what's the difference between the core and non-core file?

Reply:
We've been moving things around to establish a formal public API so the non-core version of this file is deprecated. It just makes sense to fix the path in this PR. In general, files in `core/` are safe to use in headers of downstream APIs as they won't bring in additional dependencies outside of the cuda toolkit. This isn't quite true for 1 or 2 of the files but we're working on it.